CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+CHANGES IN VERSION 1.9.0 (from 1.8.1)
+===================================
+
+Appium 1.9.0 is a feature release, comprising multiple updates.
+
+#### General
+* Full W3C Specification support.
+* Add full beta of [Espresso driver](https://github.com/appium/appium-espresso-driver)
+  for Android automation (used by specifying `automationName` capability to be
+  `"Espresso"`).
+* Add driver for [Samsung Tizen devices](https://github.com/Samsung/appium-tizen-driver)
+* Add `-image` find element strategy,
+* Fix `--async-trace` server argument, and rename as `--long-stacktrace`
+* Sample code has been moved into the main repository to aid in maintenance.
+* Fix status retrieval to speed up performance.
+
+#### iOS
+* Add support for Xcode 10 and iOS 12.
```

Comment:
I wouldn't say it is Xcode10 support. Apple might change a lot in GM build. It would be better to only mention beta.

Reply:
Even that is a lie, since scrolling no longer works, as I've noted elsewhere.